docker-compose.local.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   joplindb:
-    image: postgres
+    image: postgres:9.6
```

Comment:
dbbackup had issues with version mismatches using postgres 10 in this container, using 9 fixed it. I didn't spend too much time investigating how to get it so we had the postgres 10 client running in our joplin container with wagtail, if we find we need postgres 10 in the future that should be doable